tango_simlib/tango_yaml_tools/base.py

```diff
@@ -0,0 +1,89 @@
+#########################################################################################
+# Copyright 2020 SKA South Africa (http://ska.ac.za/)                                   #
+#                                                                                       #
+# BSD license - see LICENSE.txt for details                                             #
+#########################################################################################
+"""Module that contains the TangoToYAML class that parses a Tango device specification
+   file (xmi, fgo) or a running Tango device into YAML"""
+from pathlib import Path
```

Comment:
Is this available in Python 2.7?

Reply:
Yes, it's added in the `setup.py`, installed in Python2 and ignored in Python3